addons/auth_totp/views/res_users_views.xml

```diff
@@ -7,9 +7,9 @@
         <field name="arch" type="xml">
             <xpath expr="//search" position="inside">
                 <separator/>
-                <filter name="totp_enabled" string="Two-factor authentication Enabled" domain="[('totp_secret','!=',False)]"/>
+                <filter name="totp_enabled" string="Two-factor authentication Enabled" domain="[('totp_enabled','!=',False)]"/>
```

Comment:
This doesn't look like a stable change. `totp_secret` is no longer searchable, hence this will no longer work for database not updating their view. Maybe we should consider to add a `search` on the `totp_secret` which just support True/False.